[Doc enhancement]Update whats-new.md (#3240)
MicrosoftDocs/msteams-docs: msteams-platform/whats-new.md

---
# What's new for developers in Microsoft Teams

> Find out the new features in Microsoft Teams that are generally available (GA) or are in developer preview. You can also share your feedback. 

Reviewer: ```suggestion
Discover Microsoft Teams platform features that are generally available (GA) and in developer preview.
```
Edited for conciseness. I don't think we have to discuss sharing feedback here since it isn't the primary purpose of the topic.

> For more information on developer preview program, see [Public developer preview for Microsoft Teams](~/resources/dev-preview/developer-preview-intro.md).

Reviewer: ```suggestion
For more information, see [public developer preview for Teams](~/resources/dev-preview/developer-preview-intro.md).
```
This link is broken on my side.

> ## Teams app templates

Reviewer: ```suggestion
## Teams app template catalog
```

> Along with features, we also provide sample solutions to common team scenarios in the [**Teams app template catalog**](samples/app-templates.md). The catalog is alphabetized and the newest additions are tagged with a star ☆.

Reviewer: ```suggestion
Along with new features, we also provide [production-ready Teams app templates](samples/app-templates.md) that you can deploy right away or modify to your needs. Newly added templates are indicated with a star ☆.
```

## Submit your feedback

> We listen to the developer community for any queries, bugs, feature requests, or contributions. You can submit your feedback through any one of the [available channels](feedback.md).

Reviewer: ```suggestion
We encourage Teams developers to ask questions, file bugs, submit feature requests, and make contributions. You can submit feedback through any of the [available channels](feedback.md).
```
